core/modules/cart/helpers/totalsCacheHandler.ts

```diff
@@ -0,0 +1,19 @@
+import * as types from '../store/mutation-types'
+import { Logger } from '@vue-storefront/core/lib/logger'
+
+import { StorageManager } from '@vue-storefront/core/lib/storage-manager'
+
+export const totalsCacheHandlerFactory = (mutation, state) => {
```

Comment:
it's not a factory 🤔 Can you change name? Its misleading

Reply:
Or change it to `export const totalsCacheHandlerFactory = () => (mutation, state) => {`? `cartCacheHandlerFactory` also shouldn't be factory. We are passing there global object as parameter 😕and its not even used

Reply:
I am going to make it `cartCacheHandlerPlugin` and `totalsCacheHandlerPlugin`

Reply:
And get rid of this `(Vue) => {}` part from `cartCacheHandlerFactory`